docs/installation/kubernetes.md

```diff
@@ -0,0 +1,138 @@
+---
+id: kubernetes
+title: Kubernetes
+---
+
+## Kubernetes
+
+If you are using Kubernetes you may find the following scripts useful for a new installation of Pitchfork.
+
+They consist of a deployment, a service and a horizontal pod autoscaler that you can modify according to your needs.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pitchfork
+  labels:
+    app: pitchfork
+spec:
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+  replicas: 1
+  selector:
+    matchLabels:
+      app: pitchfork
+  template:
+    metadata:
+      labels:
+        app: pitchfork
+        release: pitchfork
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "8081"
+        prometheus.io/path: "/actuator/prometheus"
+    spec:
+      terminationGracePeriodSeconds: 0
+      containers:
+        - name: pitchfork
+          image: expediagroup/pitchfork:1.19
+          ports:
+            - containerPort: 9411
+              name: pitchfork
+            - containerPort: 8081
+              name: actuator
+          env:
+            - name: SERVER_PORT
+              value: "9411"
+            - name: JAVA_JVM_ARGS
+              value: "-Dspring.jmx.enabled=true -XX:MaxRAMPercentage=80.0"
```

Comment:
I see we've [enabled JMX](https://github.com/ExpediaGroup/pitchfork/blob/cd41d627f6c5fbedaedc520c8bb740ba3adbd90d/src/main/resources/application.yml#L72-L73) for metrics, but do we really need it?
Setting `spring.jmx.enabled` to true adds some overhead (see [here](https://github.com/spring-projects/spring-boot/issues/16090#issuecomment-469635778)) and wondering if it's useful or not.

Reply:
the kafka metrics require this to be enabled :(

Reply:
regarding the metrics, there's folks using jmx agents to get metrics.